lib/react-server-route.jsx

```diff
@@ -33,11 +31,11 @@ module.exports = function(req, res, next) {
       // but that generates HTML without any React properties, so that _would_
       // get wiped if the HTML contains a <script> element that tries to load
       // the bundle for hooking into the DOM.
-      var html = ReactDOMServer.renderToString(<RouterContext {...renderProps} />);
+      let html = ReactDOM.renderToString(<HTML>{renderToString(<RouterContext {...renderProps} />)}</HTML>);
```

Comment:
The difference between the two is just naming, and you don't need to say that's it `server` because that's how you import and where you import it from anyway `from 'react-dom/server';`